pkg/cloudflare/client.go

```diff
@@ -58,6 +59,21 @@ func (c *cf) Create(ctx context.Context, record *phonebook.DNSRecord) error {
 		Content: record.Spec.Targets[0],
 	}
 
+	// It doesn't seem the cloudflare api library has a way of supporting multiple targets
+	// I tried to create multiple entries for the same hostname in the CF dashboard and it provides an error, so I'm assuming it's not supported. Shame.
+	if len(record.Spec.Targets) > 1 {
+		log.FromContext(context.Background()).Info("Cloudflare does not support multiple targets. Selecting first entry")
```

Comment:
You can reuse the `ctx` provided as argument.

Also, now that multiple target is supported, returning an error here is the better option.

Reply:
Ah yes, my bad! that one slipped the net
will update

Reply:
Resolved in [this](https://github.com/pier-oliviert/phonebook/pull/2/commits/a998f88b7dbcd9e1921f7ce98ae5e8e2609ec9e6)  commit